Manual/HTML/Polygonal Mapping.html

```diff
@@ -0,0 +1,232 @@
+            [15, 15, 6,  42, 23, 31, 4,  12, 46, 18],
+            [41, 2,  46, 32, 5,  34, 41, 21, 18, 15],
+            [16, 25, 0,  13, 1,  37, 31, 27, 29, 33],
+            [11, 34, 20, 15, 9,  18, 10, 4,  3,  5 ],
+            [7,  8,  13, 44, 21, 12, 45, 39, 6,  7 ],
+            [28, 4,  27, 6,  4,  44, 11, 8,  16, 44]
+        ];
+        const imgIndexes = [
+            "img/Keypad/1-copyright.png",
+            "img/Keypad/2-filledstar.png",
+            "img/Keypad/3-hollowstar.png",
+            "img/Keypad/4-smileyface.png",
+            "img/Keypad/5-doublek.png",
+            "img/Keypad/6-omega.png",
+            "img/Keypad/7-squidknife.png",
+            "img/Keypad/8-pumpkin.png",
+            "img/Keypad/9-hookn.png",
+            "img/Keypad/10-teepee.png",
+            "img/Keypad/11-six.png",
+            "img/Keypad/12-squigglyn.png",
+            "img/Keypad/13-at.png",
+            "img/Keypad/14-ae.png",
+            "img/Keypad/15-meltedthree.png",
+            "img/Keypad/16-euro.png",
+            "img/Keypad/17-circle.png",
+            "img/Keypad/18-nwithhat.png",
+            "img/Keypad/19-dragon.png",
+            "img/Keypad/20-questionmark.png",
+            "img/Keypad/21-paragraph.png",
+            "img/Keypad/22-rightc.png",
+            "img/Keypad/23-leftc.png",
+            "img/Keypad/24-pitchfork.png",
+            "img/Keypad/25-tripod.png",
+            "img/Keypad/26-cursive.png",
+            "img/Keypad/27-tracks.png",
+            "img/Keypad/28-balloon.png",
+            "img/Keypad/29-weirdnose.png",
+            "img/Keypad/30-upsidedowny.png",
+            "img/Keypad/31-bt.png",
+            "img/The Cruel Modkit/ayin.png", //index 31
+            "img/The Cruel Modkit/brackets.png",
+            "img/The Cruel Modkit/ethiopic.png",
+            "img/The Cruel Modkit/katakana.png",
+            "img/The Cruel Modkit/pi.png",
+            "img/The Cruel Modkit/reversee.png",
+            "img/The Cruel Modkit/rparagraph.png",
+            "img/The Cruel Modkit/rune.png",
+            "img/Keypad Sequence/Symbol02.png",
+            "img/Keypad Sequence/Symbol14.png", //40
+            "img/Keypad Sequence/Symbol30.png",
+            "img/Keypad Sequence/Symbol41.png",
+            "img/Keypad Sequence/Symbol53.png",
+            "img/The Cruel Modkit/tamil.png",    //44
+            "img/The Cruel Modkit/theta.png",
+            "img/The Cruel Modkit/triquetra.png",
+            "img/The Cruel Modkit/yi.png",
+            "img/The Cruel Modkit/yo.png"
+        ];
+    </script>
+    <style>
+        .dark table, table {
+            margin: 1em auto;
+            border: none;
+        }
+        .table th {
+            width: 45px;
+            height: 45px;
+            padding: 0cm;
+            font-size: 25px;
+        }
+        .table img {
+            height: 45px;
+            width: 45px;
+        }
+        .dark th img {
+            filter: invert();
+        }
```

Comment:
These style classes will cause conflicts when used in the main manual. Rename these to be more specific, for example `polygonal-mapping-table`